src/setup-wp.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+sleep 30
+wp core install --url=localhost:9000 --title=DockerWP --admin_user=admin --admin_password=pass123 --admin_email=admin@mail.com --path=/var/www/wordpress --allow-root
```

Comment:
can you use same settings as we had in previous task?

Reply:
also would be nice to have this parameters in some `.env` file, so it can be easily reused